modules/dnn/src/nms.inl.hpp

```diff
@@ -101,6 +101,94 @@ inline void NMSFast_(const std::vector<BoxType>& bboxes,
     }
 }
 
+// Implement soft NMS.
+// https://arxiv.org/abs/1704.04503
```

Comment:
Please move the reference to Doxygen documentation.

Reply:
Done.